proto/debugpb.proto

```diff
@@ -148,3 +151,10 @@ message ListFailPointsResponse {
 
     repeated Entry entries = 1;
 }
+
+message GetMetricsRequest {
+}
+
+message GetMetricsResponse {
+   string metrics = 1;
```

Comment:
How about name it as `tikv`? We may dump rocksdb and gRPC metrics in the future.

Reply:
Commonly, we treat metrics as Prometheus metrics especially. Like Go, we use `/metrics` for the Prometheus handler. 

Maybe for others like RocksDB and gRPC, we can use another status or statistics API.